### PR TITLE
Beta to RC wording updates

### PR DESCRIPTION
**Pull request by crgrieve**
A  wee wording update and broken link due to it now being RC, not beta

### CODE REVIEW

File: UmbracoNetCoreUpdates.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Are you aware of some information about Umbraco on .NET 5 that isn't already add
 Please feel free to submit a Pull Request by using the **Edit this page** button at the top of this article.
 :::
 
-In this article you will find detailed instructions on [how to try out and test the current beta version of Umbraco on .NET 5](#umbraco-9-net-5-beta). You will also find a list of relevant links to official as well as unofficial resources on the upcoming release.
+In this article you will find detailed instructions on [how to try out and test the current beta version of Umbraco on .NET 5](#umbraco-9-net-5-release-candidate). You will also find a list of relevant links to official as well as unofficial resources on the upcoming release.
```
Comment (busrasengul) [inline UmbracoNetCoreUpdates.md:15]: #
```suggestion
In this article you will find detailed instructions on [how to try out and test the current release candidate version of Umbraco on .NET 5](#umbraco-9-net-5-release-candidate). You will also find a list of relevant links to official as well as unofficial resources on the upcoming release.
```

**Review by busrasengul — COMMENTED** [review]
Thought we should change the sentences for it too :)

File: UmbracoNetCoreUpdates.md (hunk 1 above)
Comment (crgrieve) [reply UmbracoNetCoreUpdates.md:15]: How did I miss that 😂?

**Comment by busrasengul** [comment]
It looks good now :) let's merge it in!

**Pull request merged**